docs/user/tutorials/vote-extensions/mitigate-frontrunning.md

```diff
@@ -0,0 +1,407 @@
+# Mitigate Auction Front-Running
+
+## What are Vote extensions?
+
+Vote extensions is arbitrary information which can be inserted into a block. This feature is part of ABCI 2.0, which is available for use in the SDK 0.50 release and part of the 0.38 CometBFT release. 
+
+More information about vote extensions can be seen [here](../../../build/building-apps/04-vote-extensions).
+
+## Context
+
+For this tutorial we are using an example of an application that is mitigating auction front-running. 
+
+Please checkout https://github.com/fatal-fruit/abci-workshop to see the example.
+
+
+## Simulation of Auction Front-Running
+
+Please checkout the branch `part-1` of the example and then run:
+
+```shell
+./scripts/single_node/setup.sh
```

Comment:
short explanation of what this script does would be useful. asking people to run scripts blindly is never good